packages/core/src/components/List/__tests__/__snapshots__/List.test.js.snap

```diff
@@ -29,7 +29,7 @@ exports[`List renders correctly with list items 1`] = `
     onFocus={[Function]}
     onMouseEnter={[Function]}
     role="option"
-    tabIndex={0}
+    tabIndex={-1}
```

Comment:
the bug already existed in the snapshot?? that shows the amount of trust you can get from this kind of testing 😓